server/mail/mail_test.go

```diff
@@ -115,7 +115,7 @@ func testSMTPSkipVerify(t *testing.T, mailer fleet.MailService) {
 	assert.Nil(t, err)
 }
 
-func testSMTPNoAuth(t *testing.T, mailer fleet.MailService) {
+func testSMTPNoAuthWithTLS(t *testing.T, mailer fleet.MailService) {
```

Comment:
Why get rid of the old test? Is there value in keeping the old `testSMTPNoAuth` and adding the new one?

Reply:
Partly a rename and also pointing the test to the new service that supports TLS.  With TLS support properly configured, we can't use our old test service (mailpit)